vb6/VisualBasic6Lexer.g4

```diff
@@ -0,0 +1,529 @@
+
+EXIT_FUNCTION: E X I T ' ' F U N C T I O N;
+
+EXIT_PROPERTY: E X I T ' ' P R O P E R T Y;
+
+EXIT_SUB: E X I T ' ' S U B;
+
+FALSE: F A L S E;
+
+FILECOPY: F I L E C O P Y;
+
+FRIEND: F R I E N D;
+
+FOR: F O R;
+
+FUNCTION: F U N C T I O N;
+
+GET: G E T;
+
+GLOBAL: G L O B A L;
+
+GOSUB: G O S U B;
+
+GOTO: G O T O;
+
+IF: I F;
+
+IMP: I M P;
+
+IMPLEMENTS: I M P L E M E N T S;
+
+IN: I N;
+
+INPUT: I N P U T;
+
+IS: I S;
+
+INTEGER: I N T E G E R;
+
+KILL: K I L L;
+
+LOAD: L O A D;
+
+LOCK: L O C K;
+
+LONG: L O N G;
+
+LOOP: L O O P;
+
+LEN: L E N;
+
+LET: L E T;
+
+LIB: L I B;
+
+LIKE: L I K E;
+
+LINE_INPUT: L I N E ' ' I N P U T;
+
+LOCK_READ: L O C K ' ' R E A D;
+
+LOCK_WRITE: L O C K ' ' W R I T E;
+
+LOCK_READ_WRITE: L O C K ' ' R E A D ' ' W R I T E;
+
+LSET: L S E T;
+
+MACRO_IF: HASH I F;
+
+MACRO_ELSEIF: HASH E L S E I F;
+
+MACRO_ELSE: HASH E L S E;
+
+MACRO_END_IF: HASH E N D ' ' I F;
+
+ME: M E;
+
+MID: M I D;
+
+MKDIR: M K D I R;
+
+MOD: M O D;
+
+NAME: N A M E;
+
+NEXT: N E X T;
+
+NEW: N E W;
+
+NOT: N O T;
+
+NOTHING: N O T H I N G;
+
+NULL: N U L L;
+
+OBJECT: O B J E C T;
+
+ON: O N;
+
+ON_ERROR: O N ' ' E R R O R;
+
+ON_LOCAL_ERROR: O N ' ' L O C A L ' ' E R R O R;
+
+OPEN: O P E N;
+
+OPTIONAL: O P T I O N A L;
+
+OPTION_BASE: O P T I O N ' ' B A S E;
+
+OPTION_EXPLICIT: O P T I O N ' ' E X P L I C I T;
+
+OPTION_COMPARE: O P T I O N ' ' C O M P A R E;
+
+OPTION_PRIVATE_MODULE:
+	O P T I O N ' ' P R I V A T E ' ' M O D U L E;
+
+OR: O R;
+
+OUTPUT: O U T P U T;
+
+PARAMARRAY: P A R A M A R R A Y;
+
+PRESERVE: P R E S E R V E;
+
+PRINT: P R I N T;
+
+PRIVATE: P R I V A T E;
+
+PROPERTY_GET: P R O P E R T Y ' ' G E T;
+
+PROPERTY_LET: P R O P E R T Y ' ' L E T;
+
+PROPERTY_SET: P R O P E R T Y ' ' S E T;
+
+PUBLIC: P U B L I C;
+
+PUT: P U T;
+
+RANDOM: R A N D O M;
+
+RANDOMIZE: R A N D O M I Z E;
+
+RAISEEVENT: R A I S E E V E N T;
+
+READ: R E A D;
+
+READ_WRITE: R E A D ' ' W R I T E;
+
+REDIM: R E D I M;
+
+REM: R E M;
+
+RESET: R E S E T;
+
+RESUME: R E S U M E;
+
+RETURN: R E T U R N;
+
+RMDIR: R M D I R;
+
+RSET: R S E T;
+
+SAVEPICTURE: S A V E P I C T U R E;
+
+SAVESETTING: S A V E S E T T I N G;
+
+SEEK: S E E K;
+
+SELECT: S E L E C T;
+
+SENDKEYS: S E N D K E Y S;
+
+SET: S E T;
+
+SETATTR: S E T A T T R;
+
+SHARED: S H A R E D;
+
+SINGLE: S I N G L E;
+
+SPC: S P C;
+
+STATIC: S T A T I C;
+
+STEP: S T E P;
+
+STOP: S T O P;
+
+STRING: S T R I N G;
+
+SUB: S U B;
+
+TAB: T A B;
+
+TEXT: T E X T;
+
+THEN: T H E N;
+
+TIME: T I M E;
+
+TO: T O;
+
+TRUE: T R U E;
+
+TYPE: T Y P E;
+
+TYPEOF: T Y P E O F;
+
+UNLOAD: U N L O A D;
+
+UNLOCK: U N L O C K;
+
+UNTIL: U N T I L;
+
+VARIANT: V A R I A N T;
+
+VERSION: V E R S I O N;
+
+WEND: W E N D;
+
+WHILE: W H I L E;
+
+WIDTH: W I D T H;
+
+WITH: W I T H;
+
+WITHEVENTS: W I T H E V E N T S;
+
+WRITE: W R I T E;
+
+XOR: X O R;
+
+// symbols
+
+AMPERSAND: '&';
+
+ASSIGN: ':=';
+
+AT: '@';
+
+COLON: ':';
+
+COMMA: ',';
+
+DIV: '\\' | '/';
```

Comment:
Integer and ordinary division have different precedence. The rule should be split:

```suggestion
DIV: '/';
IDIV: '\\';  
```

Reply:
Will commit with order change